### ISSUE TITLE
Support for empty (constant) boosters

### ISSUE THREAD

**Issue by Gufra-bollom** [jpmml/jpmml-lightgbm#41]
ERROR - Convert lightgbm model to pmml error! b'Exception in thread "main" java.lang.NullPointerException
org.jpmml.lightgbm.LightGBMUtil.parseStringArray(LightGBMUtil.java:100)
org.jpmml.lightgbm.LightGBMUtil.parseIntArray(LightGBMUtil.java:111)
org.jpmml.lightgbm.Section.getIntArray(Section.java:62)
org.jpmml.lightgbm.Tree.load(Tree.java:79)
org.jpmml.lightgbm.GBDT.load(GBDT.java:119)
org.jpmml.lightgbm.LightGBMUtil.loadGBDT(LightGBMUtil.java:53)
org.jpmml.lightgbm.LightGBMUtil.loadGBDT(LightGBMUtil.java:45)
org.jpmml.lightgbm.Main.run(Main.java:125)
org.jpmml.lightgbm.Main.main(Main.java:118)

**Comment by vruusmann** [comment]
Reproducible example about how you managed to produce a LightGBM model text file that contains a `null` value for the `right_child` attribute?

**Comment by Gufra-bollom** [comment]
> Reproducible example about how you managed to produce a LightGBM model text file that contains a `null` value for the `right_child` attribute?

 I trained a multi-class model with lightGBM with hyper-parameter "num_class" value as 6, but actual label class number in training dataset was 5, and then I got a lightGBM model with following leave information:

Tree=5
num_leaves=1
num_cat=0
split_feature=
split_gain=
threshold=
decision_type=
left_child=
right_child=
leaf_value=-34.538776391283193
leaf_count=
internal_value=
internal_count=
shrinkage=1.39804e-76

and then when I tried to transfer this model into pmml model, nullpointer exception occured.
